packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -283,6 +283,7 @@ def test_3dt_units(cluster):
         )
 
 
+@retrying.retry(wait_fixed=2000, stop_max_delay=LATENCY * 1000)
```

Comment:
We really need to fix the "we don't wait for everything to come up before reporting healthy" problem. Although yes, we can leave that out of here for now.

This is causing at least:
 - https://dcosjira.atlassian.net/browse/DCOS-340
 - https://mesosphere.atlassian.net/browse/DCOS-4183
 - https://mesosphere.atlassian.net/browse/DCOS-3229
 - https://mesosphere.atlassian.net/browse/DCOS-6168